.env

```diff
@@ -26,4 +26,5 @@ VITE_APP_POSTHOG_API_KEY=key
 
 # Settings for authentication
 PRODUCTION=false
-VITE_APP_PRODUCTION=false
\ No newline at end of file
+VITE_APP_PRODUCTION=false
+LOCAL_DEV=true
```

Comment:
Might make this a clearer name, like "STANDALONE".  This is for if you want to develop the IDE without the Cloud wrapper running.  I develop always with the cloud running and wrapping since that is how we have it in prod.  

So I don't want to run my code in production to not have it check auth stuff since I am still in development mode, but I do want to override the checks which were against the production flag.